src/book/03-guides/04-general/04-visualmesh.mdx

```diff
@@ -6,19 +6,59 @@ description: How to get started with the Visual Mesh.
 slug: /guides/general/visualmesh
 ---
 
-This guide compliments the quickstart guide on the [main Visual Mesh repo](https://github.com/Fastcode/VisualMesh) by being geared towards those with little experience of the Linux command line (i.e. bash) or of Machine Learning workflows. If you are already familiar with bash and machine learning, you may prefer to work off the Quickstart guide on the Visual Mesh repo as it is briefer than this guide.
+This guide compliments the quickstart guide on the [main Visual Mesh repo](https://github.com/Fastcode/VisualMesh) by being geared towards those with little experience of the [Linux command line (i.e. bash)](https://ubuntu.com/tutorials/command-line-for-beginners#1-overview) or of Machine Learning workflows. If you are already familiar with bash and machine learning, you may prefer to work off the Quickstart guide on the Visual Mesh repo as it is briefer than this guide.
 
-## Check that python is installed
```

Comment:
I compressed these into details so that you only get the info it it's needed. Otherwise it can just be skipped over by the user